PowerShellTools/DebugEngine/HostUi.cs

```diff
@@ -264,6 +256,58 @@ public string ReadLine(string message, string name)
             return input;
         }
 
+        public int ReadChoice(string caption, string message, IList<ChoiceItem> choices, int defaultChoice)
+        {
+            if (String.IsNullOrEmpty(caption))

```

Comment:
The guideline is using string when referring to an object and String to a class.